drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -602,6 +602,21 @@ int stm32_i2c_configure_timing(struct device *dev, uint32_t clock)
 	uint32_t presc = 1U;
 	uint32_t timing = 0U;
 
+	/* In case timing property is available, let's use it */
+	if (cfg->nr_of_timings > 0) {
+		/*  Look for adequate timing input */
+		for(uint32_t i = 0; i < (cfg->nr_of_timings); i++) {
+            uint32_t timing_speed = I2C_SPEED_GET(i2c_map_dt_bitrate(cfg->timings_list[i].i2c_speed));
```

Comment:
Indentation problem somewhere.

Reply:
hopefully fixed - checkpatch should confirm